drivers/i3c/i3c_renesas_imx3112.c

```diff
@@ -0,0 +1,416 @@
+/* renesas_imx3112_i3c.c - I3C file for Renesas I3C 1:2 bus multiplexer */
+
+/*
+ * Copyright (c) 2022 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "i3c_renesas_imx3112_regs.h"
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/i3c.h>
+#include <zephyr/drivers/i3c/ccc.h>
+#include <zephyr/drivers/i3c/devicetree.h>
+#include <zephyr/kernel.h>
+#include <zephyr/types.h>
+
+#define LOG_LEVEL CONFIG_I3C_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(i3c_renesas_imx3112);
+
+/* Number of ports that can be accessed by device */
+#define NUM_CHANNELS 2
```

Comment:
Are there higher channel count devices of the same series? If so maybe consider a
i3c_renesas_imx311x or similar name.

Reply:
There aren't